packages/playwright/src/isomorphic/teleReceiver.ts

```diff
@@ -431,8 +449,71 @@ export class TeleSuite implements reporterTypes.Suite {
     return titlePath;
   }
 
-  project(): TeleFullProject | undefined {
-    return this._project ?? this.parent?.project();
+  abstract project(): TeleFullProject | undefined;
+
+  _addTest(test: TeleTestCase) {
+    test.parent = this;
+    this._entries.push(test);
+    this.tests.push(test);
+  }
+
+  _addSuite(suite: TeleSuite) {
+    suite.parent = this;
+    this._entries.push(suite);
+    this.suites.push(suite);
+  }
+}
+
+export class TeleDescribeSuite extends TeleSuite implements reporterTypes.DescribeSuite {
+  type: 'describe' = 'describe';
+  override location!: reporterTypes.Location;
+  override parent!: TeleDescribeSuite | TeleFileSuite;
+  constructor(title: string) {
+    super(title, 'project');
```

Comment:
Let's not introduce these wrappers and only use TeleSuite. Prone to errors like this one.